### DISCUSSION TITLE
Issues when stopping/starting Mosquitto broker

### DISCUSSION THREAD

**Discussion by xhemp** [Koenkk/zigbee2mqtt#12827] [General]
Greetings! 

Have just noticed something weird  where zigbee2mqtt/bridge/state  is 'offline' after a Mosquitto MQTT broker stop and start ( very similar to https://github.com/Koenkk/zigbee2mqtt/issues/4854, which was an actual restart instead of stop/start) , only way to get it working again is either stopping and starting mosquitto a 2nd time or restarting zigbee2mqtt, this looks to be fixed back in 2020 by commit c1e5c75 but I can see it happening still with a stop and start, a few other people posted on that (already closed) issue seeing the problem.

**Comment by hmoffatt** [comment]
I also have this problem - bridge state remains offline (zigbee2mqtt/bridge/state in MQTT) after restarting zigbee2mqtt, even though zigbee2mqtt's logs shows it is connected again.

**Reply by lux73** [reply]
can't confirm the Problem - native z2m v1.28.2 works as expected
